changelog/v5.18.0.md

```diff
@@ -0,0 +1,65 @@
+- Fix CLI command `RESStorage update` (thanks @larsjohnsen)
+- Fix commentHidePersistor when using commentQuickCollapse (thanks @larsjohnsen)
+- Fix storage pruning of options with default values (thanks @larsjohnsen)
+- Hide subredditInfo for fake/reserved subreddits (thanks @prakhar1912)
+- Run migrations in the background so that they can complete before the user opens Reddit (thanks @larsjohnsen)
+- filteReddit: Filterline no longer shares state across of all /comments pages (thanks @larsjohnsen)
+- host flickr: Fix embed and use (usually) high-res link src when possible (thanks @larsjohnsen)
+- hover infocard: Clear header when loading (thanks @larsjohnsen)
+- hover: Remove type prefix in ids and fix toggling (thanks @larsjohnsen)
+- nightmode: Set background color to the `html` element to avoid FOUC in some circumstances (thanks @larsjohnsen)
+- saveComments: Don't load all saved comments on every page view, and remove indication that a comment is already saved (thanks @larsjohnsen)
+- showImages video: Restore playback when reopening expando (thanks @larsjohnsen)
+- showImages: Expandos are not marked as duplicates, but are instead marked as opened when the media is expanded (thanks @larsjohnsen)
+- showImages: Prevent Reddit overriding expando button classes in self-texts (thanks @larsjohnsen)
+- showImages: Try generic site modules when specific hosts fail (thanks @larsjohnsen)
+- subredditInfo: Fix capitalized names not being cache invalidated (thanks @larsjohnsen)
+- redesign: Add focus function to submitHelper.js (thanks @yanmahan97)
+
+### Housekeeping / Other
+
+- Clean-up (and remove many defunct) media hosts (thanks @larsjohnsen)
+- Floater: Move to utils, add support for r2 userbar and tabmenu (thanks @andytuba, @larsjohnsen)
+- Optimize generation +subscribe / +dashboard buttons to avoid potential extra reflow (thanks @larsjohnsen)
+- Optimize subredditManager (thanks @larsjohnsen)
+- Use jQuery as a direct dependency (thanks @larsjohnsen)
+- Replace `safeHtml` with direct use of DOMPurify.santize (thanks @larsjohnsen)
+- Remove jQuery dependency from utils/createElement and utils/alert (thanks @larsjohnsen)
+- Remove little-used dependency $.tokenInput (thanks @larsjohnsen)
+- Replace dependency numeral with Intl.FormatNumber (thanks @larsjohnsen)
+- Replace jquery-sortable with sortablejs (thanks @larsjohnsen)
+- Set min Firefox version to 69, and Chrome to 76 (thanks @larsjohnsen, @erikdesjardins)
+- Display update notification type depending on the last version installed (thanks @larsjohnsen)
+- dashboard: Build tabs only when activated, preventing unnecessary data from being loaded (thanks @larsjohnsen)
+- host codepen: Retrieve iframe src instead of inserting remote HTML (thanks @larsjohnsen)
+- hover: new method `watch` to invoke `begin` on `mouseover` (thanks @larsjohnsen)
+- init: Log what may prevent RES from starting (thanks @larsjohnsen)
+- ner: Remove option hideDupes (thanks @larsjohnsen)
```

Comment:
why'd we remove `hideDupes`? I feel like there's definitely people who use this?

Reply:
Right, removing duplicates is now always enabled, as it doesn't seem to be anyone who wants it otherwise.